h/static/scripts/util/url-util.js

```diff
@@ -16,7 +16,7 @@ function replaceURLParams(url, params) {
       var value = params[param];
```

Comment:
Might be worth evaluating at some point whether maintaining this is worth the effort vs just using a RFC6570-compliant URI template library.